src/test/java/unit/Group.java

```diff
@@ -221,6 +221,27 @@ void listUsersReflectsAddRemove() throws IOException {
             UFGroup.class, "group", "describe", "--name=" + name);
     assertEquals(2, groupDescribed.numMembers, "group describe shows two members");
 
+    // test the group list-user as a table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list-users", "--name=" + name);
+
+    // use regular expression testing the table format and content inside
+    assertTrue(cmd.stdErr == null || cmd.stdErr.isEmpty());
+    assertEquals(0, cmd.exitCode, "group list-user returned successfully");
+    String[] rows = cmd.stdOut.split("\\r?\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("EMAIL", rowHead[0].trim().replace("\r", ""));
+    assertEquals("POLICIES", rowHead[1].trim().replace("\r", ""));
+
+    for (int i = 1; i < rows.length; i = i + 1) {
```

Comment:
The regex matching is going to be fragile I'm afraid, for both the emails and the policy list. What you did a group list in JSON format first and then searched for those entries in the output? They should all match.

Reply:
uhhhh. if we group list in JSON. it will not test the table format as what we proposed to do in this PR. Isn't it?

Reply:
add L242 for JSON validation